pkg/cmd/pipelinerun/log_reader.go

```diff
@@ -90,12 +92,18 @@ func (lr *LogReader) readLiveLogs(pr *v1alpha1.PipelineRun) (<-chan Log, <-chan
 		}
 
 		wg.Wait()
+
+		if pr.Status.Conditions[0].Status == corev1.ConditionFalse {
+			errC <- fmt.Errorf(pr.Status.Conditions[0].Message)
+		}
 	}()
 
 	return logC, errC, nil
 }
 
 func (lr *LogReader) readAvailableLogs(pr *v1alpha1.PipelineRun) (<-chan Log, <-chan error, error) {
+	lr.waitUntilAvailable(10, 10)
```

Comment:
Can we make it configurable because 10 seconds are really less and kind of vary from each `PipelineRun`? :thinking:

Reply:
Yes, you are right, I too have observed that 10 seconds are less.

Reply:
:+1: